static/js/util/profile_edit.js

```diff
@@ -19,6 +20,49 @@ import {
 // in to whatever component it is used in.
 
 /**
+ * bind this to this.boundRadioGroupField in the constructor of a form component
+ * to update radio buttons.
+ * pass in the name (used as placeholder), key for profile, and the options.
+ *
+ * @param keySet {String[]} Path to the field
+ * @param label {String} Label for the field
+ * @param options {Object[]} A list of options for the select field
+ * @returns {ReactElement}
+ */
+export function boundRadioGroupField(keySet, label, options) {
+  const {
+    profile,
+    updateProfile
+  } = this.props;
+
+  let onChange = e => {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, e.target.value);
+    updateProfile(clone);
+  };
+
+  const selectedValue = _.get(profile, keySet);
+  const radioButtons = _.map(options, (obj) => {
```

Comment:
also when declaring an arrow function you don't need `( )` around the function arguments if there's a single argument, so this:

```js
let foo = arg => arg;
```
is equivalent to:

```js
let foo = (foo) => foo;
```
I think it looks better to leave off the parentheses if their not necessary